Userland/Libraries/LibWeb/HTML/HTMLDetailsElement.cpp

```diff
@@ -59,6 +59,13 @@ void HTMLDetailsElement::attribute_changed(FlyString const& name, Optional<Strin
 
     // https://html.spec.whatwg.org/multipage/interactive-elements.html#details-notification-task-steps
     if (name == HTML::AttributeNames::open) {
+
+        // if the open attribute was there previously and the new value is also there elide the event per spec
+        // > When the open attribute is toggled several times in succession the resulting tasks essentially get coalesced so that only one event is fired.
+        if (old_value.has_value() && value.has_value()) {
+            return;
+        }
+
         // 1. If the open attribute is added, queue a details toggle event task given the details element, "closed", and "open".
```

Comment:
Seems like it would be nicer to update step (1) to follow the current spec at https://html.spec.whatwg.org/multipage/interactive-elements.html#details-notification-task-steps

> 1. If one of oldValue or value is null and the other is not null, run the following steps, which are known as the details notification task steps, for this [details](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-details-element) element: